backend/api/api_request_classes.py

```diff
@@ -0,0 +1,7 @@
+from pydantic import Field
+
+from backend.common import models
+
+
+class CreateUserRequest(models.User):
+    password: str = Field(min_length=8, max_length=20)
```

Comment:
why do we have max length set at 20? isn't the standard 128?

Reply:
updated